src/main/scala/com/gu/support/promotions/PromoError.scala

```diff
@@ -0,0 +1,29 @@
+package com.gu.support.promotions
+
+sealed trait PromoError {
+  def msg: String
+}
+
+case object InvalidCountry extends PromoError {
```

Comment:
Minor: if we are going to display these messages to a user, I would favour referring to promo codes as 'promotions' or just 'codes'

Reply:
In fact, if these messages are going to end up being user facing we might want to ask a UXer to give them a quick review (we could amend them as a follow up to this PR if necessary?)

Reply:
Yeah good shout. I think at the moment they are remapped in subs frontend based on the error type which might be easier to manage going forward